bundles/org.openhab.ui/web/src/components/theme-switcher.vue

```diff
@@ -127,6 +136,10 @@ export default {
     setWebAudio (value) {
       localStorage.setItem('openhab.ui:webaudio.enable', (value) ? 'enabled' : 'default')
       location.reload()
+    },
+    setCommandItem (value) {
+      localStorage.setItem('openhab.ui:commandItem', value)
+      location.reload()
```

Comment:
This does not work, I guess because of the Item picker's popup.
Luckily I already have a solution:

```suggestion
      setTimeout(() => { location.reload() }, 50) // Delay reload, otherwise it doesn't work
```